db.json

```diff
@@ -5899,7 +5937,7 @@
         ],
         "beamline": "TMO",
         "creation": "Mon Jul 18 16:05:31 2022",
-        "device_class": "pcdsdevices.pim.PPM",
+        "device_class": "pcdsdevices.pim.PPMCOOL",
```

Comment:
Side note: I wonder if there's any way to avoid this problem:

1. A new class is added to pcdsdevices
2. happi is updated to expect the new class
3. All old code that referenced this device needs a pcdsdevices update or the load fails

It makes me wonder if we need to do versioning on these or have some other way to maintain backward compatibility in this sort of environment...

Reply:
The "factory class" method is nice for backwards compatibility, but I'm not a fan of the obfuscation it brings.  If I edit a class and want to check which devices in our db are affected, it's not immediately clear.  

To be clear I do think it might be the best option, nothing better comes to mind immediately.  (Other than just telling people to update...)

Reply:
I think I'm most a fan of the "device specific class" method since it's straightforward and clear- though it feels a bit strange to figure out which things belong in the specific class and which things belong in the database. Isn't part of the point of the database to include what classes devices are? Does something like a static prefix belong in a device-specific class? Does any of this even matter? I'm not sure.

Reply:
-> https://github.com/pcdshub/device_config/issues/42